corehq/apps/reports/static/reports/js/scheduled_reports_list.js

```diff
@@ -0,0 +1,185 @@
+/*
+
+    ScheduledReportModel - model representing a single row on the list / a single report filter file
+    ScheduleReportPanelModel - model representing a whole table ('My Scheduled Reports' and 'Other Scheduled Reports')
+    ScheduledReportListModel - model representing the whole page (list of reports, actions)
+
+*/
+
+hqDefine("reports/js/scheduled_reports_list", [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/components.ko', // pagination & feedback widget
+
+], function (
+    $,
+    ko,
+    _,
+) {
+
```

Comment:
good practice to declare `"use strict";` in your js files

Reply:
thanks for the tip! added!